### DISCUSSION TITLE
Colum vs. Column?

### DISCUSSION THREAD

**Discussion by josemigallas** [streetsidesoftware/cspell-dicts#3311] [General]
American English dictionaries include the biographical word `Colum`, but since cspell is case-insensitive `colum` (typo from column) is never caught.

Wouldn't it be better to remove this word from the local dictionary?

**Comment by Jason3S** [comment]
@josemigallas,

Yes, `colum` is a common mistake.

Using `flagWords`, you can always mark it as a mistake. 

Even so, this such a common mistake, that I'll remove the lower case version from the dictionaries.

PR: #3312 removes the lower case `Colum`.

**Reply by ccoVeille** [reply]
@josemigallas cc @Jason3S 

Is Colum a name then ? I'm thinking about https://github.com/streetsidesoftware/cspell-dicts/tree/main/dictionaries/people-names

**Reply by Jason3S** [reply]
@ccoVeille,

In English, it is a name. But, in other Latin based languages, it is also a word.

**Reply by Jason3S** [reply]
Please note: `Colum` is still considered valid.

<img width="321" alt="image" src="https://github.com/streetsidesoftware/cspell-dicts/assets/3740137/8aff2ff8-fa8a-4640-abc9-74770808dae9">
